src/v2/manifest/mod.rs

```diff
@@ -48,22 +52,31 @@ impl Client {
             }
         };
 
-        let mtype = match header::HeaderValue::from_str(
-            &mediatypes::MediaTypes::ManifestV2S2.to_string(),
-        ) {
-            Ok(headervalue) => headervalue,
-            Err(e) => {
-                let msg = format!("failed to parse HeaderValue from str: {}:", e);
-                error!("{}", msg);
-                return Box::new(futures::future::err::<_, _>(Error::from(msg)));
-            }
-        };
+        let accept_headers = header::HeaderMap::from_iter(
+            [
+                mediatypes::MediaTypes::ManifestV2S2,
+                mediatypes::MediaTypes::ManifestV2S1Signed,
```

Comment:
Probably want to set [`q`][1] for these so we're making our preferences clear.

[1]: https://tools.ietf.org/html/rfc7231#section-5.3.2